.github/pull_request_template.md

```diff
@@ -1,19 +1,21 @@
 ## What changes are proposed in this pull request?
  
-(Please fill in changes proposed in this fix)
+Adding a gunicorn option in build docker api in cli module in Models.
```

Comment:
Let's remove changes to this file - it's maybe a bit confusing, but you can edit your PR description (as you've done) without editing this template file :)